### PR TITLE
[Refactoring] Redundant function removal; Ternary formatting

### PR DESCRIPTION
**Pull request by ejensen**
Follow up on PR #1354

### CODE REVIEW

File: AsyncDisplayKit/ASDisplayNode.mm [hunk 1]
```diff
@@ -813,11 +813,7 @@ - (BOOL)displaysAsynchronously
 - (BOOL)_displaysAsynchronously
 {
   ASDisplayNodeAssertThreadAffinity(self);
-  if (self.isSynchronous) {
-    return NO;
-  } else {
-    return _flags.displaysAsynchronously;
-  }
+  return self.isSynchronous = NO && _flags.displaysAsynchronously;
```
Comment (nguyenhuy) [inline AsyncDisplayKit/ASDisplayNode.mm]: `self.isSynchronous = NO` looks like a minor mistake. We want to compare the value instead of assigning?

**Comment by nguyenhuy** [comment]
Besides above issue, LGTM. Thanks for working on this, @ejensen.

**Comment by ejensen** [comment]
That's embarrassing. Thanks for catching that typo @nguyenhuy

**Comment by nguyenhuy** [comment]
Not at all! We all make mistakes :)

**Comment by appleguy** [comment]
Thank you, glad you found a few other cases of the ?: !

**Pull request merged**